### PR TITLE
Bug 1830988 - Renew telemetry probes expiring in v116

### PR DESCRIPTION
**Pull request by rocketsroger**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1830988

### CODE REVIEW

**Comment by rocketsroger** [comment]
# Request for Data Collection Renewal
### Renew for 1 year
Total: 10
———
`unified_search.search_menu_tapped`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/25544

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - Currently in experimentation to go live in v115

———
`unified_search.engine_selected`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/25544

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - Currently in experimentation to go live in v115

———
`site_permissions.prompt_shown`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/25664#issuecomment-1159151213

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This measures a customer behavior. Therefore, there is no reason to support removing it at this time.

———
`site_permissions.permissions_allowed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/25664#issuecomment-1159151213

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This measures a customer behavior. Therefore, there is no reason to support removing it at this time.

———
`site_permissions.permissions_denied`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/25664#issuecomment-1159151213

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This measures a customer behavior. Therefore, there is no reason to support removing it at this time.

———
`wallpapers.wallpaper_selected`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/23382

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior and interactions with app features. Wall paper is no longer supported however we will keep probe until feature is removed from the app

———
`wallpapers.onboarding_opened`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/26893

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior and interactions with app features. Wall paper is no longer supported however we will keep probe until feature is removed from the app

———
`wallpapers.onboarding_closed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/26893

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior and interactions with app features. Wall paper is no longer supported however we will keep probe until feature is removed from the app

———
`wallpapers.onboarding_explore_more_click`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/26893

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior and interactions with app features. Wall paper is no longer supported however we will keep probe until feature is removed from the app

———
`wallpapers.learn_more_link_click`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/fenix/pull/26990

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior and interactions with app features. Wall paper is no longer supported however we will keep probe until feature is removed from the app

———

**Comment by rocketsroger** [comment]
# Request for data collection review form

All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.

1. What questions will you answer with this data?
  * How is the "Inactive tabs" feature used.
  * How often users look for/check out add-ons in settings?
  * How many have the add-on installed and enabled, over a specific time period
  * How often users interact with an installed add-on (in the toolbar menu) to look up or modify settings.

2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements? 
  * How useful is the "Inactive tabs" feature.
  * This will help us prioritize bugs and features related to WebExtensions based on user need.
 
3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
  * There are no other alternatives. 
4. Can current instrumentation answer these questions?
  * No. 
5. List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki. 
Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.

Measurement Description | Data Collection Category | Tracking Bug #
-- | -- | --
A user tapped the "Inactive tabs" header to expand the group of inactive tabs. | Category 2 - Interaction data | https://github.com/mozilla-mobile/fenix/pull/20508
A user tapped the "Inactive tabs" header to collapse the group of inactive tabs. | Category 2 - Interaction data | https://github.com/mozilla-mobile/fenix/pull/20508
A user accessed "Add-ons" from the Settings. | Category 2 - Interaction data | https://github.com/mozilla-mobile/fenix/issues/6174
A user interacted with an installed add-on in the toolbar menu. | Category 2 - Interaction data | https://github.com/mozilla-mobile/fenix/issues/6174
A list of all enabled add-ons on the device. | Category 2 - Interaction data | https://github.com/mozilla-mobile/fenix/issues/6174


6. Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
  * https://dictionary.telemetry.mozilla.org/apps/fenix 
7. How long will this data be collected? Choose one of the following:
  * I want to permanently monitor this data. cgordon@mozilla.com will monitor this over time.

8. What populations will you measure?
  * All channels, all locales, all countries 
9. If this data collection is default on, what is the opt-out mechanism for users?
  * Default Glean SDK opt-out mechanism. 
10. Please provide a general description of how you will analyze this data.
  * Glean and Glam. 
11. Where do you intend to share the results of your analysis?
  * Only on Glean, Glam, and with mobile teams. 
12. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
   * No third-party tools.

**Review by sarah541 — CHANGES_REQUESTED** [review]
Was this missed somehow?
[shared_prefs_uuid](https://github.com/mozilla-mobile/firefox-android/blob/22d398fe20df0b0d19fa3ca6de196ac74acb311c/fenix/app/metrics.yaml#LL2313C8-L2313C8)

**Comment by rocketsroger** [comment]
> Was this missed somehow? [shared_prefs_uuid](https://github.com/mozilla-mobile/firefox-android/blob/22d398fe20df0b0d19fa3ca6de196ac74acb311c/fenix/app/metrics.yaml#LL2313C8-L2313C8)

That's on purpose.  The team added this telemetry wanted it to expire in v116.  I confirmed that they don't want it to be renewed.

File: fenix/app/metrics.yaml [hunk 1]
```diff
@@ -4701,7 +4705,8 @@ tabs_tray:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-    expires: 116
+      - cgordon@mozilla.com
```
[the remark this answers is not in the document]
Comment (rocketsroger) [reply fenix/app/metrics.yaml:4708]: Good catch. I'll fix.  Thanks!

**Pull request merged**